Doc/library/socket.rst

```diff
@@ -396,10 +396,13 @@ Constants
    Many constants of these forms, documented in the Linux documentation, are
    also defined in the socket module.
 
-   .. availability:: Linux >= 2.6.25.
+   .. availability:: Linux >= 2.6.25, NetBSD >= 8.
```

Comment:
Are any of CAN_RAW_FD_FRAMES, CAN_RAW_JOIN_FILTERS, CAN_ISOTP, CAN_J1939 defined on NetBSD?

Reply:
No -- I had checked that, these symbols do not exist on NetBSD.